packages/dd-trace/src/lambda/handler.js

```diff
@@ -50,7 +55,9 @@ function crashFlush () {
       [ERROR_TYPE]: error.name
     })
   } else {
-    log.warn('An impending timeout was reached, but no root span was found. No error will be tagged.')
+    if (__isColdStart) {
+      log.warn('An impending timeout was reached, but no root span was found. No error will be tagged.')
```

Comment:
If we're having issues with this cold start feature, it's really a nice-to-have. I'd suggest removing it and making this a `log.debug` and we move forward.